inc/gpio.h

```diff
@@ -78,10 +81,10 @@ void gpio_init();
 void gpio_init_pin(GPIOAddress *address, GPIOSettings *settings);
 
 // Set the pin by address
```

Comment:
I think gpio_set_state is enough, considering you have gpio_toggle_state and gpio_get_value.